runtime/src/rent_collector.rs

```diff
@@ -117,11 +117,16 @@ impl RentCollector {
         account.saturating_sub_lamports(rent_due.lamports());
         let end_lamports = account.lamports();
 
+        let begin_account_data_len = account.data().len();
```

Comment:
Perhaps let account_data_len_reclaimed = 0. Then, in the if, set account_data_len_reclaimed=account.data().len() before zeroing it out.

Reply:
Done